### DISCUSSION TITLE
nix flake

### DISCUSSION THREAD

**Discussion by tcoopman** [arxanas/scm-record#24] [General]
The flake to run scm-diff-editor at `git-branchless` currently fails with `scm-record` no such file or directory.
I took me a while to figure out that this was extracted.

Could a flake be provided for scm-record as well, so that I can install `scm-diff-editor` via nix?

**Comment by arxanas** [comment]
I don't know anything about Nix (and didn't write the original flake) so someone else would have to provide the fix.
